src/main/java/org/apache/commons/io/input/Tailer.java

```diff
@@ -556,4 +784,181 @@ private long readLines(final RandomAccessFile reader) throws IOException {
             return rePos;
         }
     }
+
+    /**
+     * Abstraction on {@link java.io.File} which allows substitution of remote files for example using jCIFS
+     *
+     * @since 2.12.0
+     */
+    public interface Tailable {
+        /**
+         * @return  The name of the file denoted by this tailable
+         */
+        String getName();
+
+        /**
+         * @return  The full path of this tailable
+         */
+        String getPath();
```

Comment:
Confusing API name because it does not return a `Path`; either rename the API or make it return a `Path`.

Reply:
I've renamed this to `getPathName` more descriptive but now inconsistent with `java.io.File#getPath`